test/testkit/mocks/telemetrygen/telemetrygen.go

```diff
@@ -124,7 +124,7 @@ func PodSpec(signalType SignalType, opts ...Option) corev1.PodSpec {
 		Containers: []corev1.Container{
 			{
 				Name:  "telemetrygen",
-				Image: "ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.114.0",
+				Image: "ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen:v0.115.0",
```

Comment:
Can we maybe take the value from the main.go constants as well to reduce the amount of files to touch?

Reply:
I will create a proposal for using a single configuration/constant package for such values/cases